### DISCUSSION TITLE
PocketHost 0.6.0 - FTP access and static assets 🎉

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#152] [Show and tell]
[PocketHost](https://pockethost.io), the zero-config "up in 30 seconds" hosting platform for PocketBase, has reached v0.6.0 with major updates.

# Static file serving

PocketHost is now capable of serving static files such as web SPA frontends. 

For example, my app, `harvest`, has a landing page now: https://harvest.pockethost.io

<img width="495" alt="image" src="https://user-images.githubusercontent.com/1068356/210133839-556f4813-0958-4f2d-b1c3-977c0da6aa2c.png">

Static files are uploaded via the new FTP gateway described below.

# FTP Access

You can now FTP into your PocketHost instance to manage files. You can securely access `pb_data`, `pb_static`, `backups` (tgz db backups), and a new `workers` where Deno cloud workers will live (coming very soon, I have it working in a separate branch).

https://user-images.githubusercontent.com/1068356/210133878-c9e67b4b-7519-4c7e-ad64-3f09f1c0e0d7.mp4

https://user-images.githubusercontent.com/1068356/210133890-69ef627e-9d66-464a-a3ca-f56b5360c1c4.mp4

This capability is HUGE because it means users can now upload/download their db backups and frontends. While this is an alpha feature, it should work well for updating web frontends and downloading backups. Please try it out and tell me what you think!

# PocketBase *semver* management

You can now specify a semver like `^0.10.0` and PocketHost will automatically keep your instance up-to-date with the latest matching PocketBase version.

<img width="344" alt="image" src="https://user-images.githubusercontent.com/1068356/210133249-c34af855-4bce-48ba-9329-aed49af14f82.png">

PocketHost checks for and applies new PocketBase releases automatically.

# Bye bye, Docker

PocketHost no longer relies upon Docker. Instead, PocketHost now runs in a pure NodeJS environment with very few other dependencies. This decision was made because future versions of PocketHost will be installable via an npm package and will work anywhere NodeJS works.

# Stats and thanks

PocketHost growth is continuing at a pace of about 10 users per day right now. These stats are nice, but the REAL stat is that people notice immediately when PocketHost is down. That means people are really using it and beginning to rely upon it, and that's a great holiday gift to me. So THANK YOU to everyone who is trying to build something with PocketHost.

* Users: ~500
* Instances: ~700
* Instance invocations: ~51k
* Highly active instances: ~100
* New users per day: about 10
* Server:  4 GB Memory / 80 GB Disk / SFO3 - Ubuntu 22.04 x64

**Comment by osmely** [comment]
Amazing! I love this platform, I'm working on a medium-sized project and I want to thank to @benallfree  for this great software and his efforts.

**Reply by benallfree** [reply]
Say more about your project?

**Reply by osmely** [reply]
yep.. we started a short time ago, but the idea is to provide a service to create professional, company, commercial or any type of public profiles for entrepreneurs, organizations..

**Reply by benallfree** [reply]
Sounds great! I’m thinking of adding a showcase page to PocketHost, please let me know when you’re live.

**Reply by TechStudent10** [reply]
> Sounds great! I’m thinking of adding a showcase page to PocketHost, please let me know when you’re live.

May I do the same? I'm currently working on a desktop app for app building.

**Reply by benallfree** [reply]
@TechStudent10 yes, of course! When you’re ready, just post to showcase here in the discussion forum, and we can talk about how best to integrate it with the marketing page

**Reply by TechStudent10** [reply]
Alright!

**Reply by benallfree** [reply]
@TechStudent10 @osmely https://github.com/benallfree/pockethost/issues/167

**Reply by osmely** [reply]
Perfect!

**Reply by TechStudent10** [reply]
Nice! Is it fine if I post my project even though it's still in prerelease stages?

**Reply by benallfree** [reply]
@TechStudent10 Yeah please put a link in #167 👍
